src/llama.cpp

```diff
@@ -2099,6 +2101,7 @@ struct llama_hparams {
     uint32_t n_ff_shexp = 0;
     uint32_t n_expert_shared = 0;
     float    expert_weights_scale = 0.0;
+    uint32_t n_sliding = 0; // sliding window attention (SWA)
```

Comment:
```suggestion
    uint32_t n_swa = 0; // sliding window attention (SWA)
```

Reply:
Changed in https://github.com/ggerganov/llama.cpp/pull/8227/commits/ed5496fb32e9888abeaa7672aaba4d4251671457